resteasy-dependencies-bom/pom.xml

```diff
@@ -80,7 +80,7 @@
         <version.org.wildfly.core.wildfly-cli>7.0.0.Final</version.org.wildfly.core.wildfly-cli>
         <version.org.wildfly.extras.creaper>1.6.1</version.org.wildfly.extras.creaper>
         <version.org.wildfly.wildfly-arquillian-container-managed>2.2.0.Final</version.org.wildfly.wildfly-arquillian-container-managed>
-        <version.org.wildfly.wildfly-arquillian-container-remote>2.1.1.Final</version.org.wildfly.wildfly-arquillian-container-remote>
```

Comment:
is `wildfly-arquillian-container-remote` no longer used at all?

Reply:
Yes, remote was intended to be removed in RESTEASY-2466, but this is some leftover

Reply:
Marking this thread as resolved, let me know if you have some objections ...